concept/thing/impl/AttributeImpl.java

```diff
@@ -88,9 +89,13 @@ public FunctionalIterator<ThingImpl> getOwners() {
 
     @Override
     public FunctionalIterator<ThingImpl> getOwners(ThingType ownerType) {
-        return ownerType.getSubtypes().map(ot -> ((ThingTypeImpl) ot).vertex).flatMap(
-                v -> readableVertex().ins().edge(HAS, (PrefixIID.of(v.encoding().instance())), v.iid()).from()
-        ).map(ThingImpl::of);
+        return ownerType.getSubtypes().flatMap(this::getOwnersExplicit);
+    }
+
+    @Override
+    public FunctionalIterator<ThingImpl> getOwnersExplicit(ThingType ownerType) {
```

Comment:
@dmitrii-ubskii  ahh I didn't realise - we don't have the pattern of `explicit` methods on things - only Types. It doesn't make sense to have "explicit" ownership on the instance level I think?

Reply:
The idea is to get owners of this explicit type, as the `getOwners()` also gets subtypes.